notation.go

```diff
@@ -253,10 +261,10 @@ func Verify(ctx context.Context, verifier Verifier, repo registry.Repository, re
 				}
 				continue
 			}
-
 			// at this point, the signature is verified successfully. Add
 			// it to the verificationOutcomes.
 			verificationOutcomes = append(verificationOutcomes, outcome)
+			logger.Debugf("Signature verification succeeded for signature with digest %v", sigManifestDesc.Digest)
```

Comment:
IMO, we should also log the artifactDescriptor.Digest on success.

Reply:
Updated.